### PR TITLE
[monorepo] Run yarn upgrade

### PR DESCRIPTION
**Pull request by snario**
Upgrades most dependencies.

### CODE REVIEW

**Comment by snario** [comment]
Tests failing on `eslint` running out of memory.............

File: packages/e2e-tests/puppeteer/scripts/rps.ts [hunk 1]
```diff
@@ -135,3 +135,25 @@ export async function bResigns(rpsTabA: Page, rpsTabB: Page): Promise<boolean> {
...
+
+    const rpsTabA = (await browserA.pages())[0];
+    const rpsTabB = (await browserB.pages())[0];
+
+    await loadRPSApp(rpsTabA, 0);
+    await loadRPSApp(rpsTabB, 1);
+
+    await login(rpsTabA, rpsTabB);
+  }
+})();
```
Comment (geoknee) [inline packages/e2e-tests/puppeteer/scripts/rps.ts]: Perhaps this belongs in a separate PR?

Comment (snario) [reply packages/e2e-tests/puppeteer/scripts/rps.ts]: #950

File: packages/nitro-protocol/src/contract/transaction-creators/eth-asset-holder.ts [hunk 2]
```diff
@@ -10,7 +10,8 @@ import * as assetHolderTransactionCreator from './asset-holder';
 // If we don't set a gas limit some transactions will fail
 const GAS_LIMIT = 3000000;
 
-const EthAssetHolderContractInterface = new utils.Interface(EthAssetHolderArtifact.abi);
+// @ts-ignore
```
Comment (lalexgap) [inline packages/nitro-protocol/src/contract/transaction-creators/eth-asset-holder.ts]: Can we add the github issue here as well?
